lib/chatops/controller.rb

```diff
@@ -56,6 +58,15 @@ def jsonrpc_params
       params["params"] || {}
     end
 
+    def json_body
+      if request.content_type =~ %r/\Aapplication\/json\Z/i
+        hash = ActiveSupport::JSON.decode(request.raw_post) || {}
+        hash.with_indifferent_access
+      else
+        {}
```

Comment:
:bulb: There is a sneaky suability here but `{}` will answer a `Hash` but if `ActiveSupport::HashWithIndifferentAccess.new` was answered instead, you'd have a consistent type returned for both branches of the `if` statement so one always got back a `ActiveSupport::HashWithIndifferentAccess` instance (in case it needed to be updated, merged, etc. with keys that might not be consistently made of strings or symbols).